GWAS/PCA.ipynb

```diff
@@ -1049,7 +1059,7 @@
     "    # compute centroids before projecting back the samples\n",
     "    # (calculate mean/median/cov per pop)\n",
     "    if(${\"FALSE\" if homogeneous else \"TRUE\"}){\n",
```

Comment:
when pop_col is empty, homogeneous will be set to True and this loop will not happen. I guess the problem is the same in nature as with line 1004? The solution here introducing the place holder string is not elegant. But for now, let's try using `"pop"` instead of `"label_col_placeholder" ` and use `if pop_col` instead of `if pop_col is not ""`. I"ll see how it looks after we merge the changes. Perhaps I can think of better ways then.